packages/platforms/accton/x86-64/wedge100bf-32x/onlp/builds/x86_64_accton_wedge100bf_32x/module/src/platform_lib.c

```diff
@@ -31,9 +32,9 @@
 
 #define TTY_DEVICE                      "/dev/ttyACM0"
 #define TTY_PROMPT                      "@bmc:"
-#define TTY_I2C_TIMEOUT                 55000
-#define TTY_BMC_LOGIN_TIMEOUT            1000000
-#define TTY_RETRY                       10
+#define TTY_I2C_TIMEOUT                 80000
```

Comment:
Please explain in a commit message, why this value is better. Is it documented in some datasheet (name and revision).